experiment/generate_report.py

```diff
@@ -52,6 +52,12 @@ def get_arg_parser():
         action='store_true',
         default=False,
         help='If set, plots are created faster, but contain less details.')
+
+    parser.add_argument('-f',
+                        '--fuzzers',
+                        nargs='*',
+                        help='Names of the fuzzers to include in the report. '
```

Comment:
Probably fix "help='Experiment name'" above (as it allows multiple experiments).

Can remove "All of them in the experiments by default.", feels obvious.